### ISSUE TITLE
Support for the Crystal Programming Language

### ISSUE THREAD

**Issue by keplersj** [codeclimate/codeclimate-duplication#121]
The Crystal syntax is inspired by Ruby, so I believe that it shouldn't be too hard to support the language here. The mass threshold should even be the same. I'm willing to help out and contribute in any way, just tell me where to start.

**Comment by mrb** [comment]
@keplersj Hey! We're going to catch up next week so maybe we can chat about this alongside the other stuff you're working on.

**Comment by keplersj** [comment]
Woot!

**Comment by greenbigfrog** [comment]
Not a big fan of bumping issues, but has anyone been working on this?

**Comment by keplersj** [comment]
No. I'm ashamed to say I never did get to working on this. Perhaps you might be interested @greenbigfrog? As well, we have a team of people who create and maintain tooling for the Crystal programming language now @crystal-lang-tools.
